subsys/jaxrs/src/main/java/org/commonjava/indy/bind/jaxrs/keycloak/BasicAuthenticationOAuthTranslator.java

```diff
@@ -150,7 +152,7 @@ else if ( value.toLowerCase()
                     if ( authConfig.isEnabled() )
                     {
                         String encodedToken = value.substring(7);
-                        Account account = identityManager.verify( encodedToken );
+                        Account account = identityManager.verify( requestPath, encodedToken );
```

Comment:
does the 'encodedToken' contain Indy generated token? Does PNC calls "/auth/token" api to get a Indy's token and set it to http headers?

Reply:
yes, that's the agreement of the communication between builder pod and indy, to address the issue of the lifespan of SSO token, PNC will request a indy's token first for uploading requests, ref MMENG-3869 for more details.

Reply:
Do they add the build-id in the Authentication header or a new header?

Reply:
Our endpoint for uploading request requires that id in path parameter, so I just use it and no more headers required.

Reply:
My concern is Authentication header is used widely so the change may have global effect to Indy apis. e.g, some admin api may not send 'build-id' so it may fail on the validation part. And pnc needs to know how to add the build-id into the 'bearer' token (not sure how easy it is).

Reply:
Yeah, but that failure is expected, and indy can continue verify that via handling it to SSO.  Since here I am not marking it as NOT_AUTHENTICATED. 

This token is just used for uploading request, we may add path control but I just don't make it complicated since we already moved most apis out to the specific services.

Reply:
I see 'loginConfig.addFirstAuthMethod( BASIC_LOGIN_MECHANISM );'. So it will go for the basic, if fail then to keycloak. 
LGTM!

Reply:
Yeah, and it's working as above when I debug it locally. thanks.